### PR TITLE
Add External Video with video tab

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContent.jsx [hunk 1]
```diff
@@ -1,27 +1,31 @@
 import React from 'react';
 
 import TabPositionButtonGroup from '../TabPositionButtonGroup';
-import Slide from '../Slide';
+import { Button } from '../../../common';
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContent.jsx]: `import { Button } from '/imports/ui/components/common';`

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContentContainer.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const TabVideoContentContainer = ({
+  isSharingVideo,
+  stopExternalVideoShare,
+  allowExternalVideo,
+  ...props
+}) => {
+  const handleExternalVideoClick = () => {
+    const { mountModal } = props;
+    mountModal(<ExternalVideoModal />);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContentContainer.jsx:20]: empty line before

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx [hunk 3]
```diff
@@ -164,36 +166,31 @@ class VideoPlayer extends Component {
...
-    const sinceLastMessage = (timestamp - this.lastMessageTimestamp) / 1000;
-    if ((msg === 'play' && this.lastMessage === 'stop'
-         || msg === 'stop' && this.lastMessage === 'play')
-         && sinceLastMessage < THROTTLE_INTERVAL_SECONDS) {
-      return clearTimeout(this.throttleTimeout);
+  getCurrentTime() {
+    if (this.player && this.player.getCurrentTime) {
+      return Math.round(this.player.getCurrentTime());
     }
+    return null;
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx:174]: empty line before

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx [hunk 4]
```diff
@@ -212,30 +209,38 @@ class VideoPlayer extends Component {
...
-  getCurrentPlaybackRate() {
-    const intPlayer = this.player && this.player.getInternalPlayer();
+  sendSyncMessage(msg, params) {
+    const timestamp = Date.now();
 
-    return (intPlayer && intPlayer.getPlaybackRate && intPlayer.getPlaybackRate()) || 1;
-  }
+    // If message is just a quick pause/un-pause just send nothing
+    const sinceLastMessage = (timestamp - this.lastMessageTimestamp) / 1000;
+    if (
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx:223]: empty line

File: bigbluebutton-html5/imports/ui/components/external-video-player/styles.scss [hunk 5]
```diff
@@ -2,15 +2,17 @@
...
-  display: flex;
-  flex-flow: column;
-  flex-grow: 1;
-  flex-shrink: 1;
-  position: relative;
-  overflow-x: hidden;
-  overflow-y: auto;
-  border-style: none;
-  border-bottom: none;
+  // display: flex;
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/external-video-player/styles.scss]: do we need this dead styles

Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/external-video-player/styles.scss]: removed

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx (hunk 4 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx:223]: added new line

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx:174]: added line before

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContentContainer.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContentContainer.jsx:20]: added new line

File: bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContent.jsx (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/Tabs/components/TabVideoContent/TabVideoContent.jsx]: fixed

**Pull request merged**